superset/views/base.py

```diff
@@ -682,18 +674,14 @@ def apply(self, query: Query, value: Any) -> Query:
 
 
 class CsvResponse(Response):
-    """
-    Override Response to take into account csv encoding from config.py
-    """
+    """Override Response to take into account csv encoding from config.py"""
```

Comment:
Were these comments manually or automatically changed?

Reply:
Manually. If that change is not desired, I'll revert it

Reply:
@sebastianliebscher could you please revert these docstring changes.

Reply:
@john-bodley Done.